__tests__/endpoints.test.js

```diff
@@ -91,3 +91,43 @@ describe("GET /api/articles/:article_id", () => {
     })
 })
 
+describe("/api/articles", () => {
+    test("200: returns object with all the articles", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body}) => {
+            const articles = body.body
+            
+            articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(Number)
+                })
+            })
+        })
+    })
+    test("articles appear in descending order", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body}) => {
+            const articles = body.body
+            expect(articles[0].article_id).toBe(articles.length)
+        })
+    })
```

Comment:
I like the idea behind this test, but it is not a very robust test, I would recommend looking into the library jest-sorted for a nicer way to do this test assertion.